service-automation/src/com/appyschool/suiteSetup/TestSuiteSetup.java

```diff
@@ -426,23 +428,87 @@ public void testCreateStudents(HashMap<String, String> map) throws JSONException
 		map.put("$branchId", SuiteSetupProperties.getValue("BRANCH_ID_1"));
 		map.put("$branchStandardSection", branchStandardIds.get("mapId"));
 		map.put("$standard", branchStandardIds.get("standardId"));
-
 		
 		JSONObject json = CommonUtils.getJsonFromTemplate(jsonTemplate, map);
 		json.put("idNumber", "Appy" + RandomStringUtils.randomNumeric(5));
 		json.put("rollNumber", RandomStringUtils.randomNumeric(5));
```

Comment:
Why you have changed this test case?

Reply:
Because RandomStringUtils.randomNumeric(10) is same as RandomStringUtils.randomNumeric(5)+RandomStringUtils.randomNumeric(5).
Just to make the code look more clear I changed that.